ts/src/header-validator/validate-json.ts

```diff
@@ -1526,17 +1657,21 @@ function trigger(ctx: RegistrationContext, j: Json): Maybe<Trigger> {
           []
         ),
         aggregatableSourceRegistrationTime: () => aggregatableSourceRegTimeVal,
-        aggregationCoordinatorOrigin: field(
-          'aggregation_coordinator_origin',
-          aggregationCoordinatorOrigin,
-          ctx.vsv.aggregationCoordinatorOrigins[0]
-        ),
+        aggregationCoordinatorOrigin: aggregationCoordinatorOriginField,
         eventTriggerData: field('event_trigger_data', eventTriggerData, []),
         triggerContextID: field(
           'trigger_context_id',
           (ctx, j) => triggerContextID(ctx, j, aggregatableSourceRegTimeVal),
           null
         ),
+        aggregatableDebugReporting: field(
+          'aggregatable_debug_reporting',
+          (ctx, j) =>
+            struct(ctx, j, {
+              ...aggregatableDebugReportingConfig,
+            }),
```

Comment:
```suggestion
            struct(ctx, j, aggregatableDebugReportingConfig),
```
Note sure if that works though.